src/presets/iis.ts

```diff
@@ -30,7 +30,8 @@ export const iisNode = defineNitroPreset({
       await writeFile(
         resolve(nitro.options.output.dir, "index.js"),
         `
-        if (process.env.PORT.startsWith('\\')) {
+        const port = process.env.PORT
+        if (port && Number.isNaN(Number(port))) {
```

Comment:
I rather to have a specific check for `\` as it is a very specific behavior check and if another invalid value is provided we shall use explicit error

Reply:
@pi0 No, I didn't yet. Only followed up because [people reported](https://github.com/nuxt/nuxt/issues/22961#issuecomment-1867941123) the previous fix didn't work.

Reply:
It should normally work with `\` check. I have a working IIS setup in my home lab made basically for this issue! Aso wondering what is wrong with reports... Will try to check later

Reply:
FYI: @tolgahanbeyazoglu confirmed that this workaround is working as expected. But I'd be happy with a more strict one either.

Reply:
But there is even a better fix now :D